src/App.jsx

```diff
@@ -1,104 +1,23 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+const posts = postsFromServer.map(post => {
+  const user = usersFromServer.find(userData => userData.id === post.userId);
+  const comments = commentsFromServer.filter(
+    comment => comment.postId === post.id,
+  );
```

Comment:
I have a fetish for shorter syntax. Not a big deal in this case, you decide what looks better to you

```suggestion
  const user = usersFromServer.find(({ id })=> id === post.userId);
  const comments = commentsFromServer.filter(
   ({ postId }) => postId === post.id,
  );
```